docs/labs/EclipseLoot.md

```diff
@@ -84,7 +84,11 @@ It's smart about what parameters are required.
   which some people prefer to make it faster.
   To do this,
   *right-click* the project and select *Close*
+  
+    ![Closing_EclipseLoot](https://github.com/comp129/comp55/assets/148159945/7b00a40e-a86f-40e1-bcec-4bcb9fa9e8dc)
 
+        Figure: Closing Tabs and Package Explorer can help keep one's workspace clean. Not only that, but it can help
+           with being productive with classes currently being worked on and is a way to effectively close tabs.
```

Comment:
```suggestion
        > Closing Tabs and Package Explorer can help keep one's workspace clean.
        > Not only that, but it can help with being productive with classes currently being worked on
        > and is a way to effectively close tabs.
```